plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -501,7 +502,7 @@ public LocatedTableHandle getTableHandle(ConnectorSession session, SchemaTableNa
             LOG.debug("Skip %s because the table contains unsupported reader features: %s", tableName, unsupportedReaderFeatures);
             return null;
         }
-        verifySupportedColumnMapping(getColumnMappingMode(metadataEntry));
+        verifySupportedColumnMapping(getColumnMappingMode(metadataEntry, protocolEntry));
```

Comment:
Can this change go as a separate prep commit?

Reply:
Even if we separate a commit, the preparatory commit will contain almost all changes except for tests.